packages/amplify_test/lib/src/integration_test_utils/auth_cognito/integration_test_auth_utils.dart

```diff
@@ -71,77 +82,98 @@ Future<DeleteUserResponse> deleteUser(String username) async {
 /// The [verifyAttributes] flag will verify the email and phone, and should be used
 /// if tests need to bypass the verification step.
 /// The [attributes] list passes additional attributes.
-Future<AdminCreateUserResponse> adminCreateUser(
+Future<void> adminCreateUser(
```

Comment:
The response has the error in it, though, so how would we know if it failed or not?

Reply:
We throw the exception if present in the response.

Reply:
Okay, so you changed the lambda as well? Last I looked at the lambda, it sent back a successful response no matter what pretty much, it just included the errors in a field.

Reply:
@dnys1 Updated these in my personal lambda instance- I've included the changes in this PR now as well. Once the PR is approved, I'll update the instances we've already deployed.